base/setup/reactos/lang/fi-FI.rc

```diff
@@ -7,8 +7,8 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "ReactOS Asennus"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Welcome to the ReactOS Setup Wizard", IDC_STARTTITLE, 115, 8, 195, 24
-    LTEXT "Et voi asentaa ReactOS:ä suoraan tältä CDlevyltä! Käynnistä tietokoneesi tältä CD levyltä asentaaksesi ReactOS:ä.", IDC_STATIC, 115, 40, 195, 100
+    LTEXT "Tervetuloa ReactOS-asentimeen", IDC_STARTTITLE, 115, 8, 195, 24
+    LTEXT "Et voi asentaa ReactOS:ä suoraan tältä CD-levyltä! Käynnistä tietokoneesi tältä CD levyltä asentaaksesi ReactOS:ä.", IDC_STATIC, 115, 40, 195, 100
```

Comment:
@binarymaster ... New contributor. Can you check whether contributors email is accessible? Thx (Commenting here to make it pop up in the Mattermost activity stream)